account_multicompany_easy_creation/wizards/multicompany_easy_creation.xml

```diff
@@ -66,6 +65,7 @@
                 <page string="Banks">
                     <field name="bank_ids">
                         <tree editable="bottom">
+                            <field name="partner_id"/>
```

Comment:
You shouldn't present the `partner_id`. You need to put the new company's partner one.

Reply:
I set self.env.user.company_id.partner_id as the default value for the partner_id field. I did it because it doesn't matter what value you set as default, as I could understand from the class account.multicompany.bank.wiz only the acc_number field is used

Reply:
That's incorrect. You have to put the new company's partner, not the current company one.

Reply:
I forgot to state that I hid the partner_id field.
At the moment when the wizard appears there is no new company, when you click on the Accept button of the Wizard is when the new company is created based on the data established in the Wizard, so I have no way to access to the new company.
On the other hand, the partner_id field we are talking about belongs to another wizard with which the main Wizard has a One2many relationship, that is, it is a Transient model, which is not intended to be stored for a long time. 
![Captura de pantalla de 2019-10-22 23-22-29](https://user-images.githubusercontent.com/38267832/67355179-95a8b480-f524-11e9-8ea0-49f866ad51a4.png)
As i can see in de code, that other wizard is only used for the user to enter in the 'Account number' field the name of a Bank-type Journal that the new company will have when it is created. 
![Captura de pantalla de 2019-10-22 23-29-35](https://user-images.githubusercontent.com/38267832/67355207-abb67500-f524-11e9-9370-9e67d4924c81.png)
Maybe @carlosdauden can give his opinion here.

Reply:
You have been the one that has added `partner_id` field there, not @carlosdauden. Why don't you simply remove that field?

Reply:
That's what I meant in the previous comment, hehe

> I forgot to state that I hid the partner_id field.

i'm sorry, I didn't express myself well, I already removed that field, but as it is now required in v12, we must set a default value so that it does not appear in the view and there is no error when clicking on accept but I have set that value as I have previously stated.

Reply:
If you need partner_id here:
https://github.com/OCA/multi-company/pull/161/files?file-filters%5B%5D=.py&file-filters%5B%5D=.xml#diff-612c33ffbeca5455f8bb88183566b383R137
You can use self.new_company_id.partner_id.id

Reply:
I understand:
https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/res/res_bank.py#L65
https://github.com/odoo/odoo/commit/7b322d08de2b5343cdf6935a7ae4a12f1fd61958
https://github.com/odoo/odoo/blob/e1aee2aac2fd2c19dfd55734b1f51aedbc0393a6/odoo/addons/base/models/res_bank.py#L84